### ISSUE TITLE
New startup window design is not so handy as expected

### ISSUE THREAD

**Issue by laerreal** [git-cola/git-cola#1087]
After #1086 startup window became very obfuscated in some cases.
Can the new design be optional?

![before](https://user-images.githubusercontent.com/14236428/93327781-82c95d80-f823-11ea-91bf-e829d91d8d4d.png)
![after](https://user-images.githubusercontent.com/14236428/93327796-85c44e00-f823-11ea-8b77-27ca953107d2.png)

**Comment by laerreal** [comment]
It were difficult to me to revert changes because only one of `src`s is Git-Cola.

**Comment by davvid** [comment]
Thanks for catching this early before release.

Here's a few things we can do.
1) Add tooltips to disambiguate the folders entries.
2) Use `<parent_dir>/<basename>` rather than just `<basename>` as the folder name.  That would result in entries called `git-cola/src`, `cura/src`, `qemu/src`, `filesync/src`, which I think would probably make the folder view more generally useful without getting too cluttered.
3) Add a toggle switch on the left side to swap between the list and folder views.

What do you think?

**Comment by laerreal** [comment]
Looks like 3 is the best way. Let user to choose preferred design.

**Comment by davvid** [comment]
We were already using tooltips so 1) was already done.

For 2) I added logic so that it uses the longest unique path suffix instead of the basenames that it was previously using.

Lastly, there's now a tab bar at the top that allows choosing between the Folder Icon and List modes.

**Comment by laerreal** [comment]
Works fine! Thanks!

**Comment by laerreal** [comment]
Few probles still remain.
1. Some entries are refer to one path (likely, it's because of Windows understands both `/` and `\`)
2. Some pathes are not self describing (ex.: "assets" on picture; it's inside "progdefender/src")

It would be nice to optinally use full pathes in list tab.

![issue](https://user-images.githubusercontent.com/14236428/93718414-ade5e100-fb84-11ea-8460-cd5d1855f11e.png)

**Comment by davvid** [comment]
Thanks again.  I think I have this working correctly now.  I simplified it further so that the list view always displays absolute paths.  I did have to tweak it so that we always use the `/` path separator, but I think the behavior should be better now.  I haven't really tried this on Windows much so I'll dive in a bit and see if there's any other loose ends.

**Comment by laerreal** [comment]
Now it looks good (Windows). Thanks!
